ci/test/resources/build.yaml

```diff
@@ -202,7 +202,7 @@ steps:
       valueFrom: hello_image.image
     script: |
       set -ex
-      pip install ruff==0.1.11
+      pip install 'ruff>=0.11.5,<0.12'
```

Comment:
It's unfortunate that the ruff version needs to be specified in two places, but it doesn't seem trivial to fix that.